drivers/pds/lib/utils.go

```diff
@@ -481,11 +501,310 @@ func DeleteDeployment(deploymentID string) (*state.Response, error) {
 	return resp, nil
 }
 
+//GetDeploymentConnectionInfo returns the dns endpoint
+func GetDeploymentConnectionInfo(deploymentID string) (string, error) {
+	var isfound bool
+	var dnsEndpoint string
+
+	dataServiceDeployment := components.DataServiceDeployment
+	deploymentConnectionDetails, clusterDetails, err := dataServiceDeployment.GetConnectionDetails(deploymentID)
+	deploymentConnectionDetails.MarshalJSON()
+	if err != nil {
+		logrus.Errorf("An Error Occured %v", err)
+		return "", err
+	}
+	deploymentNodes := deploymentConnectionDetails.GetNodes()
+	logrus.Infof("Deployment nodes %v", deploymentNodes)
+	isfound = false
+	for key, value := range clusterDetails {
+		logrus.Infof("host details key %v value %v", key, value)
+		if strings.Contains(key, "host") || strings.Contains(key, "nodes") {
+			dnsEndpoint = fmt.Sprint(value)
+			isfound = true
+		}
+	}
+	if !isfound {
+		logrus.Errorf("No connection string found")
+		return "", err
+	}
+
+	return dnsEndpoint, nil
+}
+
+//GetDeploymentCredentials returns the password to connect to the dataservice
+func GetDeploymentCredentials(deploymentID string) (string, error) {
+	dataServiceDeployment := components.DataServiceDeployment
+	dataServicePassword, err := dataServiceDeployment.GetDeploymentCredentials(deploymentID)
+	if err != nil {
+		logrus.Errorf("An Error Occured %v", err)
+		return "", err
+	}
+	pdsPassword := dataServicePassword.GetPassword()
+	return pdsPassword, nil
+}
+
+//CreatecassandraWorkload generate workloads on the cassandra db
+func CreatecassandraWorkload(cassCommand string, deploymentName string, namespace string) (*v1.Deployment, error) {
+
+	var replicas int32 = 1
+	deploymentSpec := &v1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: deploymentName + "-",
+			Namespace:    namespace,
+			Labels:       map[string]string{"app": deploymentName},
+		},
+		Spec: v1.DeploymentSpec{
+			Replicas: &replicas,
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{"app": deploymentName},
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{"app": deploymentName},
+				},
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						{
+							Name:    deploymentName,
+							Image:   "scylladb/scylla:4.1.11",
```

Comment:
will this image always constant, if so declare as a constant

Reply:
done